packages/@coorpacademy-components/src/atom/button-link/index.js

```diff
@@ -74,14 +80,18 @@ const ButtonLink = props => {
 
   return (
     <button
+      {...(useTitle && {
+        title: ariaLabel || label
+      })}
```

Comment:
left a question for Lucas -> https://docs.google.com/document/d/1R8kWScutJ_WiN5V6h_bAvaCTBFLl7MPQmoWzeQ_ON3Q/edit